src/ajax/map_variants.php

```diff
@@ -147,6 +149,13 @@ function lovd_mapVariantToTranscripts (&$aVariant, $aTranscripts)
             foreach ($aVariantsOnTranscripts[$sVariant] as $sVariantOnTranscript) {
                 if (substr($sVariantOnTranscript, 0, strlen($aTranscript['id_ncbi'])) == $aTranscript['id_ncbi']) {
                     // Got the variant description relative to this transcript.
+                    // 2017-09-22; 3.0-20; The mappingInfo module call does not sort the positions, and as such the "start" and "end" can be in the "wrong" order.
```

Comment:
What is the reason for these type of comments:

> // 2017-09-22; 3.0-20; ...

They show removed code, but otherwise they seem superfluous given that everything is available in git.

Reply:
What do you mean "removed code"? The date and version number mentioned is indeed not really needed anymore since we have everything in git (it's merely a quick lookup/reminder of when the bugfix took place), but the comment itself is still valuable as it explains what is done below.

Reply:
Ah I linked this comment to the wrong line, sorry about that. Line 675:

> // 2017-09-22; 3.0-20; Replacing the old SOAP call to Mutalyzer with our new lovd_getVariantInfo()

This mentions removed code (old SOAP call).